### DISCUSSION TITLE
Give a short name to generated branches

### DISCUSSION THREAD

**Discussion by ikercrg** [renovatebot/renovate#16366] [Request Help]
We are using the branch names to generate our deployments. Those branch names are concatenated to some resources, in this case MongoDb indices, which means more than the limit of those indices (125 characters). 

Since the generated branch names have 70 characters, even more, we tried setting the property _branchTopic_ to "". That means that all the generated branches would have the same name (renovate), and that was important for us because in our _renovate.json_ we have set _prConcurrentLimit_ to 1. 

What we did no know was that the name was so important, and the generated branch includes all possible updates, and this is not valid anymore.

The problem is that we need short name, because with the default one, we have problems in our deployment. How can we do that?

**Comment by rarkins** [comment]
You could use the handlebars replace helper to remove some text from the branch name to make it shorter.

**Reply by ikercrg** [reply]
Thanks for the answer. Could you please give an example to set it, please?

**Reply by HonkingGoose** [reply]
Relevant Renovate docs:

- [Config template editing](https://docs.renovatebot.com/configuration-templates/)
- [Handlebars helper `replace`](https://docs.renovatebot.com/templates/#replace)

**Reply by rarkins** [reply]
Searching this repos issues and discussions should uncover some examples
